src/components/[guild]/JoinModal/components/TwitterRequirementsVerificationIssuesAlert.tsx

```diff
@@ -22,7 +22,7 @@ const TwitterRequirementsVerificationIssuesAlert = () => {
       <AlertIcon />
 
       <AlertDescription>
-        Access verification may be limited due to Twitter API issues.
+        Access verification may be limited due to X API issues.
```

Comment:
Please merge back the main branch to this one, I just removed this component, we don't need it anymore